---
source: tests/integration/snapshot.rs
expression: "zizmor().expects_failure(true).offline(true).unsetenv(\"GH_TOKEN\").args([\"pypa/sampleproject\"]).run()?"
---
fatal: no audit was performed
error: can't retrieve repository: pypa/sampleproject
 = note: try removing --offline or passing --gh-token <TOKEN>
